### PR TITLE
Add project KPI to project info

### PR DESCRIPTION
**Pull request by krzy5iek**
# Ticket

# Description
- Add project KPI to project info

### Before merge checklist
- [ ] CircleCI is passing
- [ ] Proper labels are set

### CODE REVIEW

File: app/views/project_info/show.html.haml [hunk 1]
```diff
@@ -214,3 +214,33 @@
...
+            %th Points Delivered (week)
+            %th Avg business hours in waiting for pr
+            %th Avg business hours in waiting for qa
+            %th Tickets rejected by qa
+            %th Bugs reported
+            %th Commits / dev / week
+            %th Min dev's number of commits
+            %th No of tickets added after sprint start
+        %tbody
+          - project_info[:kpi].sort_by { |kpi| kpi[:end_date] }.reverse.each do |kpi|
```
Comment (netguru-hound) [inline app/views/project_info/show.html.haml]: Line is too long. [85/80]

File: app/views/project_info/show.html.haml [hunk 2]
```diff
@@ -214,3 +214,34 @@
...
+              %th Points Delivered (week)
+              %th Avg business hours in waiting for pr
+              %th Avg business hours in waiting for qa
+              %th Tickets rejected by qa
+              %th Bugs reported
+              %th Commits / dev / week
+              %th Min dev's number of commits
+              %th No of tickets added after sprint start
+          %tbody
+            - project_info[:kpi].sort_by { |kpi| kpi[:end_date] }.reverse.each do |kpi|
```
Comment (netguru-hound) [inline app/views/project_info/show.html.haml]: Line is too long. [87/80]

File: app/views/project_info/show.html.haml [hunk 3]
```diff
@@ -214,3 +214,34 @@
...
+              %th Points Delivered (week)
+              %th Avg business hours in waiting for pr
+              %th Avg business hours in waiting for qa
+              %th Tickets rejected by qa
+              %th Bugs reported
+              %th Commits / dev / week
+              %th Min dev's number of commits
+              %th No of tickets added after sprint start
+          %tbody
+            - project_info[:kpi].sort_by { |kpi| kpi[:end_date] }.reverse.first(12).each do |kpi|
```
Comment (netguru-hound) [inline app/views/project_info/show.html.haml]: Line is too long. [97/80]

Comment (marcinjak) [inline app/views/project_info/show.html.haml]: I would consider moving this quite complicated logic out of view :)

**Comment by marcinjak** [comment]
Also please check Circle :)

File: spec/spec_helper.rb [hunk 4]
```diff
@@ -73,3 +74,5 @@
 end
 
 Capybara.javascript_driver = :selenium
+
+Capybara.save_path = ENV.fetch(‘CIRCLE_ARTIFACTS’, Rails.root.join(‘tmp/capybara’))
```
Comment (netguru-hound) [inline spec/spec_helper.rb]: Use only ascii symbols in identifiers.<br>Surrounding space missing for operator /.

File: spec/spec_helper.rb [hunk 5]
```diff
@@ -5,6 +5,7 @@
 
 require 'sucker_punch/testing/inline'
 require 'capybara/rspec'
+require ‘capybara-screenshot/rspec’
```
Comment (netguru-hound) [inline spec/spec_helper.rb]: Use only ascii symbols in identifiers.<br>Surrounding space missing for operator -.<br>Surrounding space missing for operator /.

**Pull request merged**